### PR TITLE
ISIS-3243: fix typos and formatting in simpleapp and contribute docs

### PR DESCRIPTION
**Pull request by HeVictor**
This PR fixes some of the typos and formatting issues in these docs that I've come across.

### CODE REVIEW

File: starters/adoc/modules/starters/pages/simpleapp.adoc [hunk 1]
```diff
@@ -610,9 +613,6 @@ This may be required because the application might otherwise be configured to us
 
 <.> Sets up logging to use the configuration defined in the `log4j2-test.xml` file
 
-<.> Tests typically inherit from `IsisIntegrationTestAbstract`, which provides some convenience methods to inherit from.
-In this case, the test inherits from the `IsisIntegrationTestAbstractWithFixtures` subclass which also adds in support for running fixtures.
-
 
 
 === webapp's src/main/java
```
Comment (HeVictor) [inline starters/adoc/modules/starters/pages/simpleapp.adoc:618]: Beyond a couple of typos, I rearranged the numbered annotations for `SimpleModuleIntegTestAbstract.java` in this file as I think they weren't lined up properly with the line of code they were supposed to correspond to. I made some educated guesses and assumptions here so it is possible that these might not be completely correct

Comment (andi-huber) [reply starters/adoc/modules/starters/pages/simpleapp.adoc:618]: looks good to me - thanks!

**Comment by HeVictor** [comment]
@andi-huber Thanks for reviewing my PR so quickly! Incidentally, I'm participating in [Hacktoberfest](https://hacktoberfest.com/about/) at the moment which is an event that encourages people to contribute to open-source projects in October through PR's. Would it be alright for you to add a `hacktoberfest-accepted` label to this PR to count it towards my contribution for the event?

**Comment by andi-huber** [comment]
there you go

**Pull request merged**